docs/clean-up.md

```diff
@@ -92,6 +92,33 @@ provider.
     3. Click on **Delete** at the top of the page.
     4. Follow the instructions to shut down the project.
 
+    **Close the billing account**
+
+    To remove the project from the Billing Acocunt:
+
+    1. Go to the
+       [Google Cloud Billing Console](https://console.cloud.google.com/billing){:target="\_blank"}.
+    2. Select **MY Projects**
+    3. Select **Actions** from the project list and select **Disable billing**.
+
+    To close the Billing Account you created:
+
+    1. Go to the
+       [Google Cloud Billing Console](https://console.cloud.google.com/billing){:target="\_blank"}.
+    2. Select the billing account you created.
```

Comment:
`billing account` should be capitalised everywhere for consistency or lowercased everywhere.